crates/env/src/topics.rs

```diff
@@ -201,3 +201,25 @@ pub trait Topics {
         E: Environment,
         B: TopicsBuilderBackend<E>;
 }
+
+#[doc(hidden)]
+pub struct PrefixedValue<'a, 'b, T> {
+    pub prefix: &'a [u8],
+    pub value: &'b T,
+}
```

Comment:
Even though this has hidden docs I'd love it to have some docs explaining the rationals behind its existence to a future ink! developer. ;)